repos/system_upgrade/el7toel8/actors/checkosrelease/actor.py

```diff
@@ -1,18 +1,15 @@
 from leapp.actors import Actor
 from leapp.libraries.actor.library import check_os_version, skip_check
-from leapp.libraries.common.config import version
 from leapp.models import OSReleaseFacts
 from leapp.reporting import Report
 from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
 
 
 class CheckOSRelease(Actor):
     """
-    Check if a supported release version of system's OS is in use. If not, inhibit upgrade process.
+    Check if the RHEL 7 minor version in use is supported. If not, inhibit upgrade process.
```

Comment:
Since this actor is depending on the configuration (IIRC) we should probably rephrase this even more to something like this:
```suggestion
    Check if the current RHEL minor version in use is supported. If not, inhibit upgrade process.
```

Reply:
Isn't the _"current"_ redundant to the _"in use"_ then?